### PR TITLE
fix: Add to album modal css fixes

### PR DESCRIPTION
**Pull request by y-lohse**
This PR fixes several problems with the add-to-album modal:

- The paddings, which probably conflicts with #97 and may need another update after the latest cozy-ui lands
- SIzing issues on mobile
- A thumbnail-rendering bug in chrome
- Missing borders

### CODE REVIEW

**Comment by y-lohse** [comment]
[[PHO] Régression du design de la liste des photos dans la modale de création d'albums](https://trello.com/c/oUYxkpk4/317-%5Bpho%5D-r%C3%A9gression-du-design-de-la-liste-des-photos-dans-la-modale-de-cr%C3%A9ation-d%27albums)

File: src/styles/albumsList.styl [hunk 1]
```diff
@@ -20,6 +20,7 @@
     .pho-album-photo-item
         height 4em
         width 4em
```
Comment (GoOz) [inline src/styles/albumsList.styl]: then maybe `width 4em` is not relevant anymore, is it ?

Comment (y-lohse) [inline src/styles/albumsList.styl]: I think it still is, otherwise the image will take it's natural width, won't it?

Comment (GoOz) [inline src/styles/albumsList.styl]: oh yeah right, i didn't see it was about an image.

File: src/styles/addToAlbum.styl [hunk 2]
```diff
@@ -5,10 +5,17 @@
 
     // Should be in Cozy-uy you know
     .coz-modal-section
+        box-sizing border-box
         width       440px
+        max-width 100%
+        padding 1em 0 !important
```
Comment (gregorylegarec) [inline src/styles/addToAlbum.styl]: Don't forget to remove this line when upgrading Cozy UI. :D

Comment (gregorylegarec) [inline src/styles/addToAlbum.styl]: Guys, juste update Cozy UI and use `<ModalSection />`

**Pull request merged**